cmd/axiom/ingest/ingest.go

```diff
@@ -97,11 +102,11 @@ func NewIngestCmd(f *cmdutil.Factory) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringSliceVarP(&opts.Filenames, "file", "f", nil, "File to ingest")
+	cmd.Flags().StringSliceVarP(&opts.Filenames, "file", "f", nil, "File(s) to ingest (- to read from stdin). Compressed input not supported")
```

Comment:
Since we're touching this: Why not read from stdin if no file is specified instead of requiring `-`?

Reply:
We are actually doing that already :sweat_smile: -- in a particular way.

If stdin is a tty then we require -f, if it's not then we assume stdin is a pipe or file and make -f optional.

Reply:
Could that comment be reworded to make this functionality more clear? Reads to me like I always have to pass `-f`/`--file`, even without tty

Reply:
I tried, but failed to find a concise wording :laughing: It's pretty complicated behavior and we have to fit it into 1 line :sweat_smile:

Reply:
ok, just pushed an attempt at clarifying this. Also added an example of how to upload compressed logs (to hint that it's not supported in the tool itself)